### ISSUE TITLE
How can I get a good result for a pipe SLAM?

### ISSUE THREAD

**Issue by Ali-Darzi** [gisbi-kim/FAST_LIO_SLAM#25]
Hi, 

Thanks for the good job. I have Ouster Os0-128 sensor and my goal is to have a SLAM model of a pipe in a room. While lidar sees the objects in the room, the FAST-LIO_SLAM works fine but as soon as it gets to the pipe (90cm wide, 2.2 m long with no object in it) the SLAM result is all over the place (the pipe bends, the length of it is not consistent). 
I played with the parameters and nothing good came out of it. The following result obtained by:
CONFIG:
  scan_line: 128
  blind: 0.01
  fov_degree: 90
LAUNCH:
point_filter_num: 1
filter_size_surf: 0.05
filter_size_map: 0.01
cube_side_length: 10000

<img width="1255" alt="Screen Shot 2022-05-25 at 9 52 49 PM" src="https://user-images.githubusercontent.com/69311482/170399598-59536bfa-993a-4910-b579-3dafb00d58f0.png">

Does anyone have any suggestion?

**Comment by srinivasrama** [comment]
@Ali-Darzi Do you have a bag file to share?

**Comment by Ali-Darzi** [comment]
@srinivasrama Yes, the following link is an example of recording from the pipe. 
https://drive.google.com/file/d/155Jav2F7njQcrHPdH1O-umzNpy6vPPnW/view?usp=sharing

**Comment by 85256638** [comment]
are there any updates on this  topic?
